app/helpers/flood_risk_engine/email_helper.rb

```diff
@@ -0,0 +1,20 @@
+module FloodRiskEngine
+  module EmailHelper
+
+    # Embed an image inline into html email
+    def email_image_tag(image, **options)
+      path = "/app/assets/images/#{image}"
+
+      full_path = Rails.root.join path
+
+      # This lets us resolve assets that are inside this gem
+      unless File.exist? full_path
+        full_path = "#{Gem.loaded_specs['flood_risk_engine'].full_gem_path}#{path}"
```

Comment:
I'd use `File.join` here rather than string catenation - as it's removes the need to worry about slash insertion. Also on line 6.